tests/phpunit/api/v3/ContributionPageTest.php

```diff
@@ -488,7 +486,7 @@ public function testSubmitMembershipBlockNotSeparatePaymentZeroDollarsWithEmail(
       'billing_first_name' => 'Billy',
       'billing_middle_name' => 'Goat',
       'billing_last_name' => 'Gruffier',
-      'selectMembership' => $this->_ids['membership_type'],
+      'selectMembership' => $this->_ids['membership_type'][0],
```

Comment:
@eileenmcnaughton Interesting that the tests were wrong here. That helps me with #15555 which has similar "issues".

Reply:
yep - there is so much wrong :-(